Assignment2/abczefg-Zhuo.ttl

```diff
@@ -0,0 +1,15 @@
+Placeholder to generate the folder
+
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:Class01 :includes :Sensor029 ;
+         :includes :Computer101 .
+
+:Sensor029 :hasMeasurement :Measurement8401 .
+
+:Measurement8401 :hasTemperature 29^^xsd:integer ;
```

Comment:
This will not parse. It requires the literal to be inside quotes. Same in the following line.